de/template_extending/README.md

```diff
@@ -144,4 +144,4 @@ Nur eine Sache noch. Wir müssen die beiden Templates miteinander verknüpfen. D
 
 Das war's! Probier aus, ob deine Website noch richtig funktioniert. :)
 
-> Wenn du einen Fehler `TemplateDoesNotExist` erhältst, dann heißt das, dass es gibt die Datei `blog/base.html` nicht gibt und dass `runserver` in der Konsole läuft. Halte ihn an (durch Drücken von Ctrl+C – Control und die C Taste gleichzeitig) und starte ihn neu, in dem du den Befehl `python manage.py runserver` ausführst.
\ No newline at end of file
+> Wenn du den Fehler `TemplateDoesNotExist` erhältst, dann heißt das, dass es die Datei `blog/base.html` nicht gibt und dass `runserver` in der Konsole läuft. Halte ihn an (durch Drücken von Ctrl+C – Control und die C Taste gleichzeitig) und starte ihn neu, in dem du den Befehl `python manage.py runserver` ausführst.
```

Comment:
Changed to "C-Taste" and "indem" instead of "in dem" in crowdin

Reply:
Sollte das eigentlich "Strg" sein, wie es auf deutschen (und mutmaßlich österreichischen) Tastaturen angeschrieben ist? (Da die schweizerische Tastatur für mind. 3 Sprachen funktionieren muss, ist die Taste dort wie auf dem US-Layout als "Ctrl" beschriftet.)

Reply:
(Evtl. sollten beide Abkürzungen/Beschriftungen erwähnt werden.)

Reply:
Ja, guter Punkt. Lass beides dort erwähnen.